rclpy/rclpy/impl/rcutils_logger.py

```diff
@@ -123,28 +123,24 @@ class Throttle(LoggingFilter):
 
     params = {
         'throttle_duration_sec': None,
-        'throttle_time_source_type': 'RCUTILS_STEADY_TIME',
+        'throttle_time_source_type': Clock(),
     }
 
     @classmethod
     def initialize_context(cls, context, **kwargs):
         super(Throttle, cls).initialize_context(context, **kwargs)
         context['throttle_last_logged'] = 0
-        # TODO(dhood): clarify the accepted values of the time source type, once it's actually used
-        if context['throttle_time_source_type'] != 'RCUTILS_STEADY_TIME':
+        if not isinstance(context['throttle_time_source_type'], Clock):
             raise ValueError(
                 'Received throttle_time_source_type of "{0}"; '
-                'currently only RCUTILS_STEADY_TIME is supported'
-                # TODO(dhood): remove the following note when the time source is actually rcutils
-                ' (simulated with python time.monotonic)'
+                'currently Clock object is supported.'
```

Comment:
@BMarchi nit: consider rephrasing e.g. `Received throttle_time_source_type is not a Clock instance.`